src/views/administration/notifications/Alerts.vue

```diff
@@ -134,8 +135,18 @@
                       <b-form-input id="input-3" v-model="notificationLevel" disabled class="form-control disabled" readonly trim />
                     </b-form-group>
                     <b-input-group-form-input id="input-destination" :label="$t('admin.destination')" input-group-size="mb-3"
-                                              required="true" type="text" v-model="destination" lazy="true"
+                                              :required="(!(this.alert.hasOwnProperty('teams') && this.alert.teams != null && this.alert.teams.length > 0)).toString()"
+                                              type="text" v-model="destination" lazy="true"
                                               v-debounce:750ms="updateNotificationRule" :debounce-events="'keyup'" />
+                     <b-form-group v-if="this.publisherClass === 'org.dependencytrack.notification.publisher.SendMailPublisher'"
```

Comment:
This block is currently displayed for scope `SYSTEM` too, despite addition of teams to `SYSTEM` alerts not being possible according to the API server code. 

I had a related question in the API server PR, probably makes sense to clarify that first.

Reply:
Considering this as solved, as the scope limitation has been removed on the API server side.